### PR TITLE
add missing text and remove outline fom links

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?

# Why?

# Notes

# Next steps

### CODE REVIEW

File: components/Details.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const Details: React.FunctionComponent<Props> = (props) => {
         }
 
         summary {
+          outline: none;
```
Comment (erbridge) [inline components/Details.tsx]: What motivated this change?

Comment (LBHABEBIC) [reply components/Details.tsx]: It was a request made by Gill on the ticket to remove the outline around the boxes. Does it need to be there due to accessibility standards?

Comment (erbridge) [reply components/Details.tsx]: Yeah. It's an accessibility need to highlight where the current focus is, which is what the outline is for.

**Review by erbridge — APPROVED** [review]

**Comment by erbridge** [comment]
You have two commits that should probably be squashed into one, here.

**Pull request merged**